SmallRyeReactiveMessagingIT fails intermittently

This pops up on the CI build occasionally:

```
2020-08-26 14:45:15,849 INFO  [io.quarkus] (main) camel-quarkus-integration-test-smallrye-reactive-messaging 1.1.0-SNAPSHOT native (powered by Quarkus 1.7.1.Final) started in 0.026s. Listening on: http://0.0.0.0:33629
2020-08-26 14:45:15,849 INFO  [io.quarkus] (main) Profile prod activated. 
2020-08-26 14:45:15,849 INFO  [io.quarkus] (main) Installed features: [camel-core, camel-direct, camel-file, camel-policy, camel-reactive-streams, camel-smallrye-reactive-messaging, camel-support-common, cdi, mutiny, resteasy, smallrye-context-propagation, smallrye-reactive-messaging, vertx]
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 12.015 s <<< FAILURE! - in org.apache.camel.quarkus.component.smallrye.reactive.messaging.it.SmallRyeReactiveMessagingIT
[ERROR] testSmallRyeReactiveMessagingCamelRoutePublisher  Time elapsed: 10.869 s  <<< ERROR!
org.awaitility.core.ConditionTimeoutException: Condition with org.apache.camel.quarkus.component.smallrye.reactive.messaging.it.SmallRyeReactiveMessagingTest was not fulfilled within 10 seconds.

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   SmallRyeReactiveMessagingIT>SmallRyeReactiveMessagingTest.testSmallRyeReactiveMessagingCamelRoutePublisher:62 » ConditionTimeout
[INFO] 
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0 
```

--- a/integration-tests/smallrye-reactive-messaging/src/main/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/PublisherRoute.java
+++ b/integration-tests/smallrye-reactive-messaging/src/main/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/PublisherRoute.java
@@ -26,12 +26,9 @@ import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.component.reactive.streams.api.CamelReactiveStreamsService;
 import org.eclipse.microprofile.reactive.messaging.Incoming;
 import org.eclipse.microprofile.reactive.messaging.Outgoing;
-import org.jboss.logging.Logger;
 import org.reactivestreams.Publisher;
 
 public class PublisherRoute extends RouteBuilder {
-
-    private static final Logger LOG = Logger.getLogger(PublisherRoute.class);
 
     @Inject
     CamelReactiveStreamsService camel;
@@ -41,10 +38,7 @@ public class PublisherRoute extends RouteBuilder {
 
     @Incoming("sink")
     public CompletionStage<Void> sink(String value) {
-        LOG.infof("Results before add: %s", String.join(",", results.getResults()));
-        LOG.infof("Adding result value: %s", value);
         results.addResult(value);
-        LOG.infof("Results after add: %s", String.join(",", results.getResults()));
         return CompletableFuture.completedFuture(null);
     }
 

--- a/integration-tests/smallrye-reactive-messaging/src/main/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/SmallRyeReactiveMessagingResource.java
+++ b/integration-tests/smallrye-reactive-messaging/src/main/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/SmallRyeReactiveMessagingResource.java
@@ -17,6 +17,8 @@
 package org.apache.camel.quarkus.component.smallrye.reactive.messaging.it;
 
 import java.net.URI;
+import java.util.Collections;
+import java.util.List;
 
 import javax.inject.Inject;
 import javax.ws.rs.Consumes;
@@ -52,6 +54,8 @@ public class SmallRyeReactiveMessagingResource {
     @Produces(MediaType.TEXT_PLAIN)
     @GET
     public String getValues() {
-        return String.join(",", results.getResults());
+        List<String> values = results.getResults();
+        Collections.sort(values);
+        return String.join(",", values);
     }
 }

--- a/integration-tests/smallrye-reactive-messaging/src/test/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/SmallRyeReactiveMessagingTest.java
+++ b/integration-tests/smallrye-reactive-messaging/src/test/java/org/apache/camel/quarkus/component/smallrye/reactive/messaging/it/SmallRyeReactiveMessagingTest.java
@@ -25,15 +25,12 @@ import java.util.stream.Stream;
 
 import io.quarkus.test.junit.QuarkusTest;
 import io.restassured.RestAssured;
-import org.jboss.logging.Logger;
 import org.junit.jupiter.api.Test;
 
 import static org.awaitility.Awaitility.await;
 
 @QuarkusTest
 class SmallRyeReactiveMessagingTest {
-
-    private static final Logger LOG = Logger.getLogger(SmallRyeReactiveMessagingTest.class);
 
     @Test
     public void testSmallRyeReactiveMessagingCamelRouteSubscriber() {
@@ -69,7 +66,6 @@ class SmallRyeReactiveMessagingTest {
                     .extract()
                     .body()
                     .asString();
-            LOG.infof("Values: %s", response);
             return response.equals("A,B,C,D");
         });
     }

